docs/en/sql-reference/table-functions/s3Cluster.md

```diff
@@ -27,18 +27,21 @@ A table with the specified structure for reading or writing data in the specifie
 
 **Examples**
 
-Select the data from all files in the cluster `cluster_simple`:
```

Comment:
Lol, how this sentence passed the review...

Reply:
I thought it was a bit off... 😄  Thanks!